MekHQ/src/mekhq/campaign/Campaign.java

```diff
@@ -2660,6 +2660,11 @@ public int getDeploymentDeficit(AtBContract contract) {
         } else if (contract.getStartDate().after(getDate())) {
             // Contracts in the future don't have deficits...yet.
             return 0;
+        } else if (contract.getStartDate().equals(getDate())) {
```

Comment:
This can be simplified by combining it with the previous as `if (!getDate().after(contract.getStartDate())`. For that matter, all three conditions in a row could be combined into a single statement.

Reply:
Changed to use compareTo